configs/example.ini

```diff
@@ -22,6 +22,11 @@ db = postgresql+psycopg2:///pajbot?options=-c%%20search_path%%3Dpajbot1_streamer
 ; set this to 1 if your bot is a verified bot (increased rate limits) on Twitch
 ; More info about verified bots can be found here: https://dev.twitch.tv/docs/irc/guide#known-and-verified-bots
 ;verified = 1
+; if your bot is known set known to 1
+;known = 1
+; if you wish to change how whipers are managed. setting whisper_output to 0 will disable whispers, 
+; setting it to 2 will output all whsipers in chat. The default setting 1 is whispers act as whispers.
+;whisper_output = 1
```

Comment:
I'd prefer if this was a string-based setting, e.g. `whispers = disabled/ratelimited/unlimited`